src/Server.ts

```diff
@@ -5,23 +5,29 @@ import { getExpressAppConfig } from 'utils/ConfigLoader';
 
 import { createContainer } from 'app/ContainerFactory';
 import 'app/controllers/index';
+import errorHandlers from 'app/middleware/ErrorHandler';
```

Comment:
I'd prefer you to import specific error handlers and use it explicitly e.g.

```
import { notFoundHandler, defaultHandler } from 'app/middleware/ErrorHandler';
...
use(notFoundHandler, defaultHandler)
```

IMO it will be easier to understand what is happening.